kong/runloop/handler.lua

```diff
@@ -626,7 +627,7 @@ do
 
   local function get_now_ms()
```

Comment:
Can you rename this to:
```suggestion
  local function get_monotonic_ms()
```

Reply:
done